backend/requirements.txt

```diff
@@ -1,90 +1,92 @@
-appdirs==1.4.3
-asgiref==3.2.10
-asn1crypto==1.4.0
-attrs==19.3.0
-CacheControl==0.12.6
-certifi==2020.6.20
-cffi==1.15.0
-chardet==3.0.4
-codecov==2.1.8
-colorama==0.4.3
-contextlib2==0.6.0
+appdirs==1.4.4
+asgiref==3.7.2
+asn1crypto==1.5.1
+attrs==23.1.0
+CacheControl==0.13.1
+certifi==2023.5.7
+cffi==1.15.1
+chardet==5.1.0
+charset-normalizer==3.1.0
+codecov==2.1.13
+colorama==0.4.6
+contextlib2==21.6.0
 coreapi==2.3.3
 coreschema==0.0.4
-coverage==5.2.1
-cryptography==3.3.2
-defusedxml==0.6.0
-diff-match-patch==20200713
-distlib==0.3.0
-distro==1.4.0
-Django==3.1
+coverage==7.2.7
+cryptography==41.0.1
+defusedxml==0.7.1
+diff-match-patch==20230430
+distlib==0.3.6
+distro==1.8.0
+Django==4.2.2
```

Comment:
Ooolala ;)

Reply:
yeah the Django upgrade might come with some changes ... The warnings you are seeing are likely a new requirement that they are phasing in ...